docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,278 @@
+---
+title: Getting Started with F# in Visual Studio Code with Ionide
+description: Learn how to write and run F# code in Visual Studio Code with Ionide 
+keywords: visual f#, f#, functional programming, .NET, Visual Studio Code, vscode, Ionide
+author: cartermp
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: devlang-fsharp
+ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
+---
+
 # Getting Started with F# in Visual Studio Code with Ionide
 
-> [!NOTE]
-This is still in-progress.
\ No newline at end of file
+You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a great cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.  Visit [Ionide.io](http://ionide.io) to learn more about the plugin.
+
+## Prerequisites
+
+F# 4.0 or higher must be installed on your machine to use Ionide.
+
+### Windows
+
+If you're on Windows, the easiest way to do that is to install [Visual Studio with the Visual F# Tools](getting-started-visual-studio.md#installing-the-visual-f#-tools).  This will install all of the necessary components to write and execute F# code.
```

Comment:
Since the focus of this doc is using F# with ionide, I wouldn't say installing visual studio with F# is the easiest way to get started on windows.  it takes a god awful long time to install VS especially when it decides to create a system restore point.  It's easier to install all of the prereqs on their own, because chances are the reader didn't/won't check  off the ones they can during their VS installation.

You should make note that if you already have VS installed then adding F# through the VS installer is the best way to go so you get all the language features in VS, and even if you have VS + F# installed already, the subsequent instructions are still relevant and must be followed to get ionide working properly on your system.

Maybe add a note on how to check that all the prereqs were properly installed? (maybe this note goes in the troubleshooting section?)